src/index.ts

```diff
@@ -1,7 +1,8 @@
 export * from "./governor.js";
 export * from "./votes.js";
 export * from "./contract_result.js";
-
+export * from "./contract_error.js";
+export { nativeToScVal } from "@stellar/stellar-sdk";
```

Comment:
am OK with letting this workaround go by, but part of me thinks we should try transforming the incoming `Array<any>` into:

```
arg: string[]

fn {
    real_arg = arg.map // map string to xdr.ScVal
}
```

That way users of the SDK have a clear parameter type (we probs need to add a comment explaining what kind of string it is)